src/chart/generateCategoricalChart.tsx

```diff
@@ -1027,6 +1035,50 @@ export const generateCategoricalChart = ({
       if (!_.isNil(this.props.syncId)) {
         this.addListener();
       }
+
+      this.accessibilityManager.setDetails({
+        container: this.container,
+        offset: {
+          left: this.props.margin.left ?? 0,
+          top: this.props.margin.top ?? 0,
+        },
+        coordinateList: this.state.tooltipTicks,
+        mouseHandlerCallback: this.handleMouseMove,
+        layout: this.props.layout,
+      });
+    }
+
+    getSnapshotBeforeUpdate(
```

Comment:
just a callout that using this means not refactoring this to a more functional component any time soon. Probably weren't going to anyways, but `getSnapshotBeforeUpdate` doesn't exist in the world of hooks

Reply:
@nikolasrieble just tagging you for visibility

Reply:
Admittedly I do not think this makes a large difference in context of an overall complex component that will be hard to take apart. We might consider simply rewriting it from scratch in a functional way, and use the existing component as a comparison / test.

Reply:
Sweet, sounds good